containers/server-bundle/Dockerfile.nodejava.s390x

```diff
@@ -1,14 +1,13 @@
-FROM s390x/debian:stretch
```

Comment:
FROM s390x/alpine?

Reply:
i'll look into it for a future PR, too many changes at once here.